iterations_runners/iterations_runner.som

```diff
@@ -0,0 +1,131 @@
+BenchmarkHarness = (
+
+    | benchmarkClass numIterations param debug instrument numCores |
+
+    krunInit = primitive
+    krunDone = primitive
+    krunGetNumCores = primitive
+    krunMeasure: idx = primitive
+    krunGetWallclock: idx = primitive
+    krunGetCoreCyclesDouble: index core: core = primitive
+
+    run: args = (
+        self processArguments: args.
+        self runBenchmark.
+    )
+
+    usage = (
+      'Iterations runner for SOM VMs.' println.
```

Comment:
I *think* you can put an extra '\n' at the end of the strings if you want two blank lines? I haven't checked that in a while though...

Reply:
Ah cool, fixed: 35571d607cd13bb6c2f505424637621707105fa6